.github/workflows/on_pull_request.yml

```diff
@@ -59,58 +59,11 @@ jobs:
             packages/paste-design-tokens/types/**/*
             packages/paste-codemods/tools/.cache/mappings.json
 
-  type_check:
-    name: Type check
+  build_website:
```

Comment:
ℹ️ Job that will build the website immediately after the other package assets are finished building. This will then be saved to the cache to be used by the test job later.